index.html

```diff
@@ -664,15 +668,9 @@ <h2 id="img_dialog_title"></h2>
       },
 
       showPage: function(page, tab) {
-        /*if (tab != 'default') {
-          var tpl = Polymer.dom(this.$.pages).querySelector('.tpl.' + tab);
-          if (!tpl) {
-            var error = Polymer.dom(this.$.pages).querySelector('#not-found');
-            error.page = tab;
-            this.set('routeData.tab', 'not-found');
-          }
-        }*/
-
+        /* If the requested page matches the current tab, then return true
+         * so the <template dom-if> chunks will DOM stamp the wrapped element
+         * into the site. */
         return page == tab;
```

Comment:
Removed the commented out code and added a comment describing what `showPage` does.